fabric_plugin/tunnel.py

```diff
@@ -42,22 +39,27 @@ def accept(channel, (src_addr, src_port), (dest_addr, dest_port)):
         try:
             sock.connect((local_host, local_port))
         except Exception as e:
+            try:
+                channel.close()
+            except Exception as e2:
+                close_error = ' (While trying to close channel: {0})'.format(
+                    e2)
+            else:
+                close_error = ''
```

Comment:
Don't you at least want to provide a generic "Unknown" here? If we can't close the channel, we might want to state that we didn't close it.